### DISCUSSION TITLE
When I am the owner of a repository, I am showing up twice.

### DISCUSSION THREAD

**Discussion by jim4067** [o2sh/onefetch#1020] [Q&A] [answered]
I have noticed that in repos where only one person has been making contributions, it shows up twice. 

What could be causing this? 

![Screenshot from 2023-04-19 21-43-05](https://user-images.githubusercontent.com/30603522/233170708-d02f0557-771b-4f6c-90bb-75647c0e7ca0.png)

![image](https://user-images.githubusercontent.com/30603522/233170642-f9abba8d-90e1-49e3-b3e7-e567b0fa51ec.png)

**Comment by spenserblack** [comment, marked as answer]
Does this resolve your issue? https://github.com/o2sh/onefetch/issues/923
